apps/dashboard/src/lib/utils/services/announcement/index.ts

```diff
@@ -0,0 +1,93 @@
+import type { Announcement } from '$lib/utils/types/anouncement';
```

Comment:
@Chifez I thought you mentioned you have changed everything from announcement to newsfeed?

This file has announcement, this line and the types file has announcement.

Please go over everything again.